recipe/meta.yaml

```
@@ -93,6 +93,7 @@ requirements:
     - pydocstyle >=6.3.0,<6.4.0
     - pyflakes >=3.2.0,<3.3.0
     - pylint >=3.1,<4
+    - pyqt5-sip<12.16 #[py==3.8]
```

```suggestion
    - pyqt5-sip <12.16  # [py==3.8]
```
additional whitespaces